index.html

```diff
@@ -28,6 +27,132 @@ <h1>Hemerson Foreste</h1>
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>
+  
+            <div class="project-division">
+              <img src="images/Laptop-Left.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>
+  
+            <div class="project-division">
+              <img src="images/Laptop-Pink.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>          
+        </section>
+  
+        <!--About Section-->
+        <section id="about">
+            <h2>About</h2>
+            <p class="about-content">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+              labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+            </p>
+            <button type="button"><span>Get my CV</span><span><img class="arrow-download" src="images/ic_download.svg" alt="arrow download"></span></button>
+            <div id="Languages">
+                <h3>Languages</h3>
+              <ul>
+                  <li>JavaScript(ES5/ES6)</li>
+                  <li>Ruby</li>
+                  <li>HTML</li>
+                  <li>CSS</li>
+              </ul>
+              <img id="img-languages" src="images/diese.png" alt="#">
+            </div>
+  
+            <div id="Frameworks">
+                <h3>Frameworks</h3>
+              <ul>
+                  <li>Bootstrap</li>
+                  <li>Ruby on rails</li>
+                  <li>RSpec</li>
+                  <li>Capibara</li>
+                  <li>Selenium</li>
+              </ul>
+              <img id="img-frameworks"  src="images/ic_framework.svg" alt="framework">
```

Comment:
- [ ] Instead of an image please use a background image. Decoration should be background images not `img` tags. **_You should apply it to other decoration images in the about section._**